src/modules/esl-utils/misc/test/array.test.ts

```diff
@@ -48,6 +48,75 @@ describe('misc/array helper tests', () => {
 
 type V = string | number;
 
+describe('unwrap', () => {
+  document.body.innerHTML = '<div>1</div><div>2</div>';
+
+  test('NodeList to unwrap into first Node', () => {
+    expect(unwrap(document.querySelectorAll('div'))).toEqual(document.querySelector('div'));
+  });
+
+  test.each([
```

Comment:
Could you please reformat test cases to a single lines to improve readability of the test